requirements.txt

```diff
@@ -5,3 +5,4 @@ pbr
 termcolor==1.1.0
 setproctitle
 ordereddict==1.1
+ujson
```

Comment:
>  It's kinda logic to not have pip in production because most of time you have to compile something when installing from pip and gcc is not really recommended is production environments.

it's only a system design decision. cause then you say rolling release distributions are not production usable they are used also in production. 

I was telling there are solutions above, here it is :
Now, with the latest pip version (get-pip) there can have binary wheels pip packages built so that you don't need to compile anything on the production server ..
am I wrong ?